tests/timeseries/test_logical_meter.py

```diff
@@ -427,14 +377,14 @@ async def test_consumer_power_2_grid_meters(
         self,
         mocker: MockerFixture,
     ) -> None:
-        """Test the grid power formula with grid meters."""
+        """Test the grid power formula with two grid meters."""
         mockgrid = MockMicrogrid(grid_meter=False)
         # with no further sucessor these will be detected as grid meters
         mockgrid.add_consumer_meters(2)
         await mockgrid.start(mocker)
 
         logical_meter = microgrid.logical_meter()
-        grid_consumption_recv = logical_meter.grid_consumption_power.new_receiver()
+        grid_consumption_recv = logical_meter.grid_power.new_receiver()
```

Comment:
I guess you are purposely not using `consumption` (with the new operator) anymore here because this is already tested in the new operator tests, right?